From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 25 Dec 2018 22:38:13 +0000
Subject: tools/include/.../bitsperlong.h: Add support for riscv

tools/include/uapi/asm/bitsperlong.h needs to include the right
architecture's <asm/bitsperlong.h> for any architecture that may be
64-bit.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/tools/include/uapi/asm/bitsperlong.h
+++ b/tools/include/uapi/asm/bitsperlong.h
@@ -13,6 +13,8 @@
 #include "../../arch/mips/include/uapi/asm/bitsperlong.h"
 #elif defined(__ia64__)
 #include "../../arch/ia64/include/uapi/asm/bitsperlong.h"
+#elif defined(__riscv)
+#include "../../arch/riscv/include/uapi/asm/bitsperlong.h"
 #else
 #include <asm-generic/bitsperlong.h>
 #endif
